src/Sulu/Bundle/AdminBundle/Resources/js/containers/Application/tests/Application.test.js

```diff
@@ -4,7 +4,25 @@ import {render, mount} from 'enzyme';
 import Application from '../Application';
 import Router from '../../../services/Router';
 
-jest.mock('../../../services/Router', () => function() {});
+jest.mock('../../../services/Router', () => function() {
```

Comment:
Just use a `jest.fn` in these cases, then we can assert for stuff later if we need to and it would currently fit into one line:
```javascript
jest.mock('...', () => jest.fn());
```